LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentNonce.kt

```diff
@@ -0,0 +1,121 @@
+package com.braintreepayments.api.localpayment
+
+import com.braintreepayments.api.core.PaymentMethodNonce
+import com.braintreepayments.api.core.PostalAddress
+import com.braintreepayments.api.core.PostalAddressParser.fromJson
+import com.braintreepayments.api.sharedutils.Json
+import kotlinx.parcelize.Parcelize
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * [PaymentMethodNonce] representing a local payment.
+ *
+ * @see PaymentMethodNonce
+ *
+ * @property clientMetadataId The ClientMetadataId associated with this transaction.
+ * @property billingAddress The billing address of the user if requested with additional scopes.
+ * @property shippingAddress The shipping address of the user provided by checkout flows.
+ * @property givenName The first name associated with the local payment.
+ * @property surname The last name associated with the local payment.
+ * @property phone The phone number associated with the local payment.
+ * @property email The email address associated with this local payment
+ * @property payerId The Payer ID provided in local payment flows.
+ */
+@Parcelize
+class LocalPaymentNonce internal constructor(
+    override val string: String,
+    override val isDefault: Boolean,
+    val clientMetadataId: String?,
+    val billingAddress: PostalAddress,
+    val shippingAddress: PostalAddress,
+    val givenName: String,
+    val surname: String,
+    val phone: String,
+    val email: String?,
+    val payerId: String,
+    private val type: String?
```

Comment:
This value was not used or exposed. Should we remove it?

Reply:
Yeah it doesn't seem like it's needed. I think it's a good idea to remove.